components/XOGame.js

```diff
@@ -0,0 +1,140 @@
+import styled from "styled-components";
+import {useState} from "react";
+
+const players = {
+
+  HUMAN: {
+  SYM: "X",
+  NAME: "You",
+  },
+};
+
+export default function XOGame() {
+  
+  const [board, setBoard] = useState([
+    ["", "", ""],
+    ["", "", ""],
+    ["", "", ""],
+  ]);
+
+  function playFieldNr(arrayIndex, index) {
+    
+    
+    board[arrayIndex][index] = players?.HUMAN?.SYM;
+    setBoard((board) => [...board]);
+    
+    
+  }
+
+  function playAgainFieldNr() {
+    setBoard([
+      ["", "", ""],
+      ["", "", ""],
+      ["", "", ""],
+    ]);
+   
+  }
+
+  
+  return (
+      <>
+        <Matchfield>
+          <Col />
+            <Cell onClick={() => playFieldNr(0, 0)} >
+              {board[0][0]}
+            </Cell>
+            <Cell onClick={() => playFieldNr(0, 1)} >
+              {board[0][1]}
+            </Cell>
+            <Cell onClick={() => playFieldNr(0, 2)} >
+              {board[0][2]}
+            </Cell>
+          
+          <Col />
+            <Cell onClick={() => playFieldNr(1, 0)} >
+              {board[1][0]}
+            </Cell>
+            <Cell onClick={() => playFieldNr(1, 1)} >
+              {board[1][1]}
+            </Cell>
+            <Cell onClick={() => playFieldNr(1, 2)} >
+              {board[1][2]}
+            </Cell>
+          
+          <Col />
+            <Cell onClick={() => playFieldNr(2, 0)} >
+              {board[2][0]}
+            </Cell>
+            <Cell onClick={() => playFieldNr(2, 1)} >
+              {board[2][1]}
+            </Cell>
+            <Cell onClick={() => playFieldNr(2, 2)} >
+              {board[2][2]}
+            </Cell>
+          
+        </Matchfield>
+        
+        <Button onClick={playAgainFieldNr}>
+            Restart
+        </Button>
+          
+        
+      </>
+      
+    );
+}
+
+const Matchfield = styled.section`
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row;`is the default property when setting `dislay: flex;`
You can remove that line of code